app/views/content/funding-and-support/tuition-fee-and-maintenance-loans.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Tuition fee and maintenance loans"
+heading: "Tuition fee and maintenance loans"
+description: |-
+    Find out how much money you could get to help pay your fees and living expenses while you train.
+related_content:
+    Who do you want to teach?: "/train-to-be-a-teacher/who-do-you-want-to-teach"
+    Improve your subject knowledge: "/train-to-be-a-teacher/subject-knowledge-enhancement"
+    Salaries and benefits of teaching: "/salaries-and-benefits"
```

Comment:
```suggestion
    Salaries and benefits: "/salaries-and-benefits"
```
I think 'of teaching' sounds a bit weird as it doesn't go with 'salaries'; also inconsistent with link text on bursaries page